src/serlo-editor-repo/plugin-text/components/text-editor.tsx

```diff
@@ -162,6 +162,9 @@ export function TextEditor(props: TextEditorProps) {
   }
 
   function handleEditableKeyDown(event: React.KeyboardEvent) {
+    // stop slate from loosing focus when using esc
+    if (event.key === 'Escape') return false
```

Comment:
> The only quickfix that I can make right now is that we blur completely on Escape.

Hej @hejtful,
following you comment I tried this for testing and it improved my keyboard editing experience a lot actually. Do you know of any cases where this might cause problems?

Reply:
(moved it further down so it should only stop the list plugin from stealing focus)

Reply:
I don't see any problems with this, it's actually a super smooth solution, and I'm ashamed I didn't think of it myself 😄

Reply:
Nice! Don't worry, you found all the pieces :)